static/js/settings_bots.js

```diff
@@ -127,6 +129,7 @@ exports.set_up = function () {
             var full_name = $('#create_bot_name').val();
             var short_name = $('#create_bot_short_name').val() || $('#create_bot_short_name').text();
             var payload_url = $('#create_payload_url').val();
+            var interface_type = $('#create_interface_type :selected').val();
```

Comment:
I think I added it here by mistake, it is useful when there are multiple options selected.